### PR TITLE
updated license to GPLv3

### LINKED ISSUE #60: consistent license
As noted by @IAlibay in comments on PR #59 , there's an inconsistency in the licensing:
- docs say GPL v3 (see PR #59)
- LICENSE file says MIT
- setup.py says MIT 

Obviously, this needs to be consistent.

The choice of license leads us into hotly debated territory that requires lawyers (at least in principle). MDAnalysis itself is licensed under "GPL v2 or any later version". This means that GPL v3 is an allowed license.

@ojeda-e you need to set your license.

### PR DESCRIPTION
**Pull request by ojeda-e**
## Description

Fixes #60 

- [x] License updated: changed MIT for GPLv3.
- [x] License updated in `setup.py`.

### CODE REVIEW

**Review by orbeckst — APPROVED** [review]
LGTM, just see the one comment.

File: LICENSE [hunk 1]
```diff
@@ -1,22 +1,574 @@
...
+copy of the Program in return for a fee.
+
+END OF TERMS AND CONDITIONS
+
+How to Apply These Terms to Your New Programs If you develop a new program, and
+you want it to be of the greatest possible use to the public, the best way to
+achieve this is to make it free software which everyone can redistribute and
+change under these terms.
+
+To do so, attach the following notices to the program. It is safest to attach
```
Comment (orbeckst) [inline LICENSE:530]: I suggest you include the example wording in your init file. In particular, the part that allows selecting a later version of the license.

**Review by orbeckst — APPROVED** [review]
Lgtm, just see the one comment as a suggestion.

File: LICENSE (hunk 1 above)
Comment (ojeda-e) [reply LICENSE:530]: Thanks, added.

**Pull request merged**